### PR TITLE
Fixup name of SystemServiceProcessServiceControllerPackageVersion

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: build/dependencies.props [hunk 1]
```diff
@@ -41,7 +41,7 @@
     <SerilogSinksFilePackageVersion>3.2.0</SerilogSinksFilePackageVersion>
     <SystemDiagnosticsDiagnosticSourcePackageVersion>4.4.1</SystemDiagnosticsDiagnosticSourcePackageVersion>
     <SystemReflectionMetadataPackageVersion>1.5.0</SystemReflectionMetadataPackageVersion>
-    <SystemServiceProcessServiceControllerVersion>4.5.0-preview1-25914-04</SystemServiceProcessServiceControllerVersion>
+    <SystemServiceProcessServiceControllerPackageVersion>4.5.0-preview1-25922-03</SystemServiceProcessServiceControllerPackageVersion>
```
Comment (pranavkm) [inline build/dependencies.props]: @JunTaoLuo the mirrored version is slightly newer.

**Comment by pranavkm** [comment]
The package was missing. Uploaded it and Travis passed.

**Comment by Tratcher** [comment]
Missing from where? It's supposed to be on nuget.org.

**Comment by pranavkm** [comment]
I picked up the version that https://github.com/dotnet/versions/blob/master/build-info/dotnet/corefx/master/Latest_Packages.txt pointed to (or was pointing to yesterday). It's slightly higher than the one on NuGet.org and wasn't available on our feeds.

**Pull request merged**